cypress/tests/integration/planters/[planterid].cy.js

```diff
@@ -0,0 +1,92 @@
+import planter from '../../../../doc/examples/planters/940.json';
+import tree186734 from '../../../fixtures/tree186734.json';
+
+const organization = {
+  id: 1,
+  name: '180Earth',
+  area: 'Shirimatunda',
+  country: 'Tanzania',
+  created_at: 'November 11, 2019',
+  about:
+    'Greenway is a Youth-Driven Environmental Protection Organization providing alternative solutions to single-use plastic and planting carbon-sucking trees for socio-economic development and reducing climate crisis. Our social work includes reforestation, landscape restoration, climate education, awareness campaign, conducting research, outreach activities, and collaborating with key stakeholders to implement sustainable solutions.',
+  mission:
+    'To combat climate change, desertification, land degradation, carbon emission by inspiring healthier communities affected by severe climate disorder and modestly reducing pollution by 2050.',
+};
+
+beforeEach(() => {
+  Cypress.env('nock') && cy.task('clearNock'); // This will clear any mocks that have been set
+});
+
+describe('Planter page', () => {
+  it('getServerSideProps return mocks', () => {
+    const path = `/planters/${planter.id}`;
+
+    Cypress.env('nock') &&
+      cy.task('nocks', {
+        hostname: 'http://127.0.0.1:4010/mock',
+        routes: [
+          {
+            method: 'GET',
+            path: `/planters/${planter.id}`,
+            statusCode: 200,
+            body: {
+              ...planter,
+            },
+          },
+          {
+            method: 'GET',
+            path: '/trees/featured',
+            statusCode: 200,
+            body: {
+              trees: [{ ...tree186734 }],
+            },
+          },
+          {
+            method: 'GET',
+            path: `/species/query/planter_id=940`,
+            statusCode: 200,
+            body: {
+              species: [
+                {
+                  ...tree186734,
+                },
+              ],
+            },
+          },
+          {
+            method: 'GET',
+            path: `/organizations/query/planter_id=940`,
+            statusCode: 200,
+            body: [{ ...organization }],
+          },
+          {
+            method: 'GET',
+            path: `/trees/query/planter_id=940&limit=4`,
+            statusCode: 200,
+            body: {
+              trees: [
+                {
+                  ...planter,
+                },
+              ],
+            },
+          },
+        ],
+      });
+
+    cy.visit(path, {
+      failOnStatusCode: false,
+    });
+    cy.contains(planter.id);
+  });
+
+  it('should render planters/{planterId} page and display the name of the planter', () => {
+    const path = `/planters/${planter.id}`;
+
+    cy.intercept(path, { doc: { example: '940.json' } });
+
+    cy.request(path);
+
+    cy.get('.MuiTypography-h2').contains(/sebastian gaertner/i);
+  });
```

Comment:
This test is not passing with `npm run cy:nock`. It seems a redundant with the first test though. I think this test can be removed.

` cy.get('.MuiTypography-h2').contains(/sebastian gaertner/i);`
This assertion can be used in the first test to improve it.

Reply:
@QuaidBartolomei, I removed it, I didn't notice that. Thank you.